### ISSUE TITLE
Can't install because of an issue with "opencollective" package.

### ISSUE THREAD

**Issue by lubomir-chatoryiskyi** [akiran/react-slick#1003]
Hello!
I'm trying to install "react-slick" but install process breaks down. 
An error "ENOENT: no such file or directory, chmod '/node_modules/opencollective/dist/bin/opencollective.js'" occurs.
Environment: Docker container, Ubuntu16, x64.
Thanks for any advice.

**Comment by laveesingh** [comment]
Could you try and reproduce it on codesandbox?

**Comment by lubomir-chatoryiskyi** [comment]
I've tried to remove "react-slick" dependency on codesandbox and to add it again - it works. But I'm not sure how does it work on codesandbox. Also, I've tried to install "react-slick" on another environment - MacOS, it works. Seems something wrong with the environment.

**Comment by arif-hanif** [comment]
getting same error

**Comment by laveesingh** [comment]
I managed to successfully install it in docker environment without any ado, also in ubuntu 16, x64. Following is the very basic associated Dockerfile.
```Dockerfile
FROM node

ENV NPM_CONFIG_LOGLEVEL warn
ARG app_env
ENV APP_ENV $app_env

RUN mkdir -p /frontend
WORKDIR /frontend
COPY ./frontend ./

RUN npm install

CMD if [ ${APP_ENV} = production ]; \
	then \
	npm install -g http-server && \
	npm run build && \
	cd build && \
	hs -p 3000; \
	else \
	npm run start; \
	fi

EXPOSE 3000
```

**Comment by ajuni7** [comment]
Receiving the same issue on my end as well. Is it not possible to remove this package?

Similar to: https://github.com/storybooks/storybook/issues/1937

**Comment by ajuni7** [comment]
Also I tried it with Node.js v6.12.3 and it seems to work. Not sure why it fails with future versions.

**Comment by lubomir-chatoryiskyi** [comment]
My node version is v8.7.0 and it fails.

**Comment by akiran** [comment]
Can you give setup to replicate this issue, so that we can debug it and fix it

**Comment by kurtinatlanta** [comment]
We get this error on our build box with (with opencollective 1.03, I think - I don't have shell access to these systems):
```
> react-slick@0.18.0 postinstall <redacted>/node_modules/react-slick
> opencollective postinstall

sh: <redacted>/node_modules/.bin/opencollective: Permission denied
```
We don't get this problem with any of our other dependencies.

**Comment by akiran** [comment]
Removed opencollective for now and released 0.18.1

We will add it back after investigating the issue

**Comment by kurtinatlanta** [comment]
Thanks. Greatly appreciated.
